.changeset/warm-otters-explode.md

```diff
@@ -0,0 +1,40 @@
+---
+'@norges-domstoler/dds-components': minor
+---
+
+Legger til støtte for å bygge `<OverflowMenu>` med subkomponenter (lenke, knapp, divider, header) som barn. Dette gjør `<OverflowMenu>` sitt bruksområde mer generelt.
+
+Før:
+```jsx
+<OverflowMenu
+  userProps={{
+    name: 'Agata',
+    href: '/min-profil',
+  }}
+  items={[
+    {
+      title: 'Saker',
+      href: '/saker',
+    }
+  ]}
+/>
+```'
```

Comment:
Fnutten bak backtickene må nok bort, måtte ha noe der for at dette ikke skulle telles som avsluttende for suggestion 😂 

Samme med den nedenfor

Reply:
Så det, fikser 😄